### DISCUSSION TITLE
Best way to get trending, popular, etc. shows and movies including translations?

### DISCUSSION THREAD

**Discussion by henrikfroehling** [trakt/trakt-api#281] [Q&A] [answered]
There is an API endpoint to get the translations for a single show or movie.

But is there also a possibility to get whole lists of shows or movies including the translations without calling "get translations" for each single show or movie?

I know, it's possible with the TMDb API, but is this also possible with the Trakt API?

**Comment by rectifyer** [comment, marked as answer]
Not currently possible. We don’t support that for caching reasons since it would make the API calls basically uncacheable with such specific parameters.
